recipes/_lustre_install.rb

```diff
@@ -49,7 +49,7 @@
   end
 
   kernel_module 'lnet'
-elsif node['platform'] == 'centos' && node['platform_version'].split('.')[1].to_i == 7
+elsif node['platform'] == 'centos' && node['platform_version'].split('.')[1].to_i >= 7
```

Comment:
minor - Please change the log below
"Chef::Log.warn("Unsupported version of Centos, #{node['platform_version']}, supported versions are 7.5, 7.6 and 7.7")"

Reply:
done